__tests__/notifications/index.ts

```diff
@@ -1253,6 +1258,158 @@ describe('storeNotificationBundle', () => {
     ]);
   });
 
+  it('should generate source_post_submitted notification', async () => {
```

Comment:
Realized we were missing these 3 tests, hence, added here now along with the fix.